physionet-django/user/templates/user/edit_emails.html

```diff
@@ -91,15 +91,19 @@ <h4>Public Email</h4>
 
 <hr>
 <h4>Add Email</h4>
-<p>Add a new email address.</p>
+<p>Add a new email address(Max emails: {{ max_associated_emails_allowed }}).</p>
```

Comment:
Missing space here. `Add a new email address(Max emails:` should be `Add a new email address (Max emails:`